entrypoint.sh

```diff
@@ -73,25 +73,29 @@ isBranchBlocked() {
         return 0
     fi
 
-    # target branch is considered blocked if its current HEAD is ahead of the next branch HEAD
-    # ie the head of the 'target' branch is not an ancester of the 'after target' branch
+    target branch is considered blocked if its current HEAD is ahead of the next branch HEAD
+    ie the head of the 'target' branch is not an ancester of the 'after target' branch
```

Comment:
I think the comments have been uncommented by mistake.

Reply:
Ah, block unquote as I was forcing failures. Thanks!